spec/features/admin/admin_authors_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe 'Admin Authors' do
+  before do
+    create(:author)
+
+    admin_user = create(:admin_user)
+    login_as(admin_user, scope: :admin_user)
+
+    visit admin_authors_path
+  end
+
+  let(:author_attributes) { attributes_for(:author) }
+
+  it 'Admin can create an Author' do
+    click_link('New Author')
+
+    expect(page).to have_current_path(new_admin_author_path)
+
+    fill_in 'author[first_name]', with: author_attributes[:first_name]
+    fill_in 'author[last_name]', with: author_attributes[:last_name]
+
+    expect { click_button('Create Author'); sleep(1) }.to change { Author.count }.by(1)
```

Comment:
Do you really need `sleep` there?
Can you rewrite test and check that after creation you will be redirected to show page where title will be the name of Author.last?

Reply:
Why do you mark this comment as resolved and do nothing?

Reply:
Sorry, I've changed it in next PR